src/StateService/Network/StateRoot.cs

```diff
@@ -0,0 +1,131 @@
+using Neo.Cryptography;
+using Neo.Cryptography.ECC;
+using Neo.IO;
+using Neo.IO.Json;
+using Neo.Ledger;
+using Neo.Network.P2P;
+using Neo.Network.P2P.Payloads;
+using Neo.Persistence;
+using Neo.SmartContract;
+using Neo.SmartContract.Native;
+using System;
+using System.IO;
+
+namespace Neo.Plugins.StateService.Network
+{
+    public class StateRoot : ICloneable<StateRoot>, IVerifiable, ISerializable
```

Comment:
Why `Network`?

Reply:
It's a p2p payload like block.

Reply:
Are you going to use neo-project/neo#2101?

Reply:
Yes. `StateRoot` is serialized as data part.